Makefile

```diff
@@ -113,10 +113,14 @@ DOCKER_PLATFORM=--platform $(BUILD_PLATFORM)
 endif
 
 # Code instrumentation
+debug?=no
+relwithdebinfo?=no
```

Comment:
Please use `relwithdbginfo` or `relwithdebuginfo`.
`deb` is sometimes used for`debian` and may cause confusion.

Reply:
The compile mode names are borrowed from CMake, where it have `Release`, `Debug,` `RelWithDebInfo` and `MinSizRel` https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html . Although we don't have to follow it, I thought using the name that is already being used would be less surprise. Also this name has already being used before this commit.

Reply:
ok, following some naming convention seems better than trying to avoid this confusion.